### DISCUSSION TITLE
How to increase the function description length limit in .NET semantic kernel?

### DISCUSSION THREAD

**Discussion by sjadlakha** [microsoft/semantic-kernel#7172] [Q&A] [answered]
I ran into a problem:
Microsoft.SemanticKernel.HttpOperationException: Invalid 'tools[9].function.desceiption': string too long. Expected a string with maximum length 1024, but got a string with 3114 length instead.

One of the functions in my plugins has a detailed description because of which this exception is raised.

Is there a way to increase this limit ?

I'm on .net semantic kernel 1.13.0

**Comment by madsbolaris** [comment, marked as answer]
This exception is being thrown by the underlying model service (I'm assuming you're using OpenAI or Azure OpenAI), so unfortunately this isn't something that Semantic Kernel can fix. If you need a larger description, you may fare better by adding this context to the system prompt.
